openapi/p4pa-pu-sil.openapi.yaml

```diff
@@ -325,7 +325,7 @@ paths:
         content:
           application/json:
             schema:
-              $ref: "#/components/schemas/ExportRequestDTO"
+              $ref: 'PaidExportRequest'
```

Comment:
@serdimic PaidExportRequest and ClassificationsExportRequest requires the SIL to know the organizationId and the debtPositionTypeOrgId...
Do you think we should hide them to it?